File: app/src/main/java/com/irontec/axier/rxweather/model/Weather.java
package com.irontec.axier.rxweather.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by axier on 14/7/17.
 */

public class Weather {

    @SerializedName("location")
    private Location location;
    @SerializedName("current")
    private Current current;
    @SerializedName("forecast")
    private Forecast forecast;

    public Weather() {
    }

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

    public Current getCurrent() {
        return current;
    }

    public void setCurrent(Current current) {
        this.current = current;
    }

    public Forecast getForecast() {
        return forecast;
    }

    public void setForecast(Forecast forecast) {
        this.forecast = forecast;
    }
}

File: app/src/main/java/com/irontec/axier/rxweather/model/Weather.kt
package com.irontec.axier.rxweather.model

import com.google.gson.annotations.SerializedName

/**
 * Created by axier on 14/7/17.
 */

class Weather {

    @SerializedName("location")
    var location: Location? = null
    @SerializedName("current")
    var current: Current? = null
    @SerializedName("forecast")
    var forecast: Forecast? = null
}
